hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/flow/txn/WorkDone.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.workflows.handle.flow.txn;
+
+public enum WorkDone {
```

Comment:
Also feels overkill?

Reply:
We need to easily differentiate where we track fee payments and where we track transaction for network utilization. Did not see other readable option. Do you have any suggestion ?